packages/apollo-link-state/package.json

```diff
@@ -79,7 +79,8 @@
   },
   "dependencies": {
     "apollo-utilities": "^1.0.6",
-    "graphql-anywhere": "^4.1.0-alpha.0"
+    "graphql-anywhere": "^4.1.0-alpha.0",
+    "lodash": "^4.17.5"
```

Comment:
If merge is the only thing you're using, you could instead depend on 'lodash.merge'.

Reply:
@majelbstoat I tried lodash.merge. ES2015 style import didn't work with it.  It is only working only if I use require statement

Reply:
Lodash & the TS compiler are weird sometimes 😢 

Try `import merge = require('lodash/merge');` for the submodule and `npm i @types/lodash -D` so we have the proper typings. This is how we do it in `react-apollo`.